TwitchLib.Client.Enums/TwitchLib.Client.Enums.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
+    <PackageId>TwitchLib.Client.Enums</PackageId>
+    <Version>2.1.3</Version>
+    <Description>Client component of TwitchLib. This component allows you to access Twitch chat and whispers, as well as the events that are sent over this connection.</Description>
```

Comment:
Probably should have a description relevant to package, especially if we plan to upload the package.